src/components/Cart/Cart.js

```diff
@@ -8,49 +8,59 @@ const Cart = ({ isCartVisible, closeCart, content, removeFromCart }) => {
 
   return (
     <div
-      data-testid='cart-test'
-      className={`${styles.cart} ${
-        isCartVisible ? styles.visible : styles.invisible
+      className={`${styles['cart-container']} ${
+        !isCartVisible ? styles['no-pointer'] : ''
       }`}
     >
-      <div className={styles.header}>
-        <div className={styles['added-games']}>
-          {content.length === 0
-            ? 'No games yet'
-            : content.length === 1
-            ? `${content.length} game`
-            : `${content.length} games`}
-        </div>
-        <div
-          className={styles['close-button']}
-          onClick={closeCart}
-          data-testid='close-cart-test'
-        >
-          ✖
+      <div
+        data-testid='cart-test'
+        className={`${styles.cart} ${
+          isCartVisible ? styles.visible : styles.invisible
```

Comment:
you can use a simple library like `classnames` to make these `className` concatenations more readable.